stellarphot/settings/models.py

```diff
@@ -702,10 +702,11 @@ class PhotometryOptionalSettings(BaseModelWithTableRep):
     include_dig_noise: Annotated[
         bool,
         Field(
+            title="Include Digitization Noise",
             description=(
                 "Should the digitization noise be included in the calculation of "
                 "the noise for each observation?"
-            )
+            ),
```

Comment:
Is this comma causing the build to fail? It doesnt seem like the comma is needed, but I could be wrong.

Reply:
No, the issue is that the code doesn't currently check for titles that have been custom set in the pydantic model. `ruff`, the code linter we use, insists on a trailing comma on the last item. Either with or without comma is syntactically correct, but having it there makes it a bit harder to forget about the comma if you add a new item. At least I think that is the rationale...

Reply:
Should be fixed now